DataFormats/Detectors/PHOS/include/DataFormatsPHOS/EventHandler.h

```diff
@@ -329,14 +346,15 @@ class EventHandler
   TriggerRange mTriggerRecordsCellIndices; ///< trigger record for cluster cell index type
   TriggerRange mTriggerRecordsCells;       ///< Trigger record for cell type
 
-  ClusterRange mClusters;             /// container for clusters in timeframe
-  CellIndexRange mClusterCellIndices; /// container for cell indices in timeframe
-  CellRange mCells;                   /// container for cells in timeframe
+  ClusterRange mClusters;                                                            /// container for clusters in timeframe
+  CellIndexRange mClusterCellIndices;                                                /// container for cell indices in timeframe
+  CellRange mCells;                                                                  /// container for cells in timeframe
+  const o2::dataformats::MCTruthContainer<o2::phos::MCLabel>* mCellLabels = nullptr; /// pointer to the MCTruthContainer for cells in timeframe
 
   ClassDefNV(EventHandler, 1);
```

Comment:
The new data member mCellLabels requires ClassDefNV to be incremented.

Reply:
done